docs/Reference/HTTP2.md

```diff
@@ -2,7 +2,7 @@
 
 ## HTTP2
 
-_Fastify_ offers **experimental support** for HTTP2 starting from Node 8 LTS,
+_Fastify_ supports HTTP2 starting from Node 8 LTS,
 which includes HTTP2 without a flag; HTTP2 is supported over either HTTPS or
 plaintext.
```

Comment:
```suggestion
_Fastify_ supports HTTP2 over either HTTPS or plaintext (h2).
```

I would simplify the full sentence

Reply:
Simplifying sounds good. But don't you mean `HTTPS (h2) or plaintext (h2c)` instead?

Reply:
Good spot, I meant `h2c`.